src/azure-cli/azure/cli/command_modules/config/custom.py

```diff
@@ -86,6 +91,7 @@ def config_unset(cmd, key=None, local=False):
 
         with ScopedConfig(cmd.cli_ctx.config, local):
             cmd.cli_ctx.config.remove_option(section, name)
+    telemetry.set_debug_info('ConfigUnset', ' '.join(key))
```

Comment:
If user unset a property, why not just remove the property from Telemetry？

Reply:
To record user behavior of the unset command.